platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -119,4 +129,34 @@ public void init() throws Exception {
 
         return export(Paths.get(exportDirectory));
     }
+
+    BundleContext getBundleContext() {
+        return FrameworkUtil.getBundle(ConfigurationMigrationManager.class)
+                .getBundleContext();
+    }
+
+    private Collection<MigrationWarning> exportMigratables(Path exportDirectory) {
+        List<MigrationWarning> warnings = new LinkedList<>();
+
+        for (ServiceReference<Migratable> serviceReference : getMigratableServiceReferences()) {
+
+            Migratable migratable = getBundleContext().getService(serviceReference);
+            MigrationMetadata migrationMetadata =
+                    migratable.export(exportDirectory.resolve((String) serviceReference.getProperty(
+                            "service.pid")));
```

Comment:
Should we use a constant for service.pid? https://osgi.org/javadoc/r2/org/osgi/framework/Constants.html#SERVICE_PID